### PR TITLE
JFG-1075(1076): Fix report generation

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java [hunk 1]
```diff
@@ -33,9 +33,9 @@ public ReportingRestController(DynamicReportingService dynamicReportingService)
         this.dynamicReportingRestController = new DynamicReportingRestController(dynamicReportingService);
     }
     
-    @GetMapping(value = "/")
+    @GetMapping(value = "")
     public WebAsyncTask<ResponseEntity<Resource>> getReport(
-            @RequestParam(name = "sessionId", required = true) String sessionId,
+            @RequestParam(name = "sessionId") String sessionId,
```
Comment (dlatnikov) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java]: why is it optional now?

Comment (anton-antonenko) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java]: it's not optional. it's required by default, so parameter is unneeded.

File: jaas/pom.xml [hunk 2]
```diff
@@ -249,7 +249,6 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
             <version>2.4</version>
```
Comment (a-badaev) [inline jaas/pom.xml]: I think the version can be missed and will be controlled and in sync with Spring Boot version

Comment (anton-antonenko) [inline jaas/pom.xml]: Unfortunately, it's not working. We must specify the version. I can leave this one, or use the one from boot dependencies - 2.9.5

**Pull request merged**